enterprise/server/remote_execution/containers/firecracker/firecracker.go

```diff
@@ -1273,10 +1273,11 @@ func (c *FirecrackerContainer) hotSwapWorkspace(ctx context.Context, execClient
 			return status.InternalErrorf("failed to stub out workspace drive: %s", err)
 		}
 		if c.workspaceVBD != nil {
-			if err := c.workspaceVBD.Unmount(ctx); err != nil {
+			err := c.workspaceVBD.Unmount(ctx)
```

Comment:
nice catch! do you mind doing this for the other unmounts as well?

Reply:
unmountAllVBDs doesn't exit early if there's an error, so we should be all good https://github.com/buildbuddy-io/buildbuddy/blob/master/enterprise/server/remote_execution/containers/firecracker/firecracker.go#L2399

Reply:
ah cool, thanks for checking :+1: